scientific_library/tvb/simulator/monitors.py

```diff
@@ -54,6 +54,7 @@
 
 """
 
+import os
```

Comment:
why do monitors need access to `os` module? please remove

Reply:
ok